extensions/dacpac/src/test/testDacFxService.ts

```diff
@@ -170,4 +170,7 @@ export class DacFxTestService implements mssql.IDacFxService {
 		};
 		return Promise.resolve(streamingJobValidationResult);
 	}
+	parseTSqlScript(filePath: string, databaseSchemaProvider: string): Thenable<mssql.ParseTSqlScriptResult> {
```

Comment:
does this Thenable also need to be changed to Promise to match the function in DacFxService.ts?